script/bootstrap.bat

```diff
@@ -0,0 +1,27 @@
+@REM Copyright (C) 2021 Stephen G. Tuggy
+
+@REM https://github.com/vegastrike/Vega-Strike-Engine-Source
+
+@REM This file is part of Vega Strike.
+
+@REM Vega Strike is free software: you can redistribute it and/or modify
+@REM it under the terms of the GNU General Public License as published by
+@REM the Free Software Foundation, either version 3 of the License, or
+@REM (at your option) any later version.
+
+@REM Vega Strike is distributed in the hope that it will be useful,
+@REM but WITHOUT ANY WARRANTY; without even the implied warranty of
+@REM MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+@REM GNU General Public License for more details.
+
+@REM You should have received a copy of the GNU General Public License
+@REM along with Vega Strike.  If not, see <https://www.gnu.org/licenses/>.
+
+mkdir C:\Projects
```

Comment:
This should be variable that folks can define that defaults to `C:/Projects` (BTW, using a forward Slash on Windows prevents needing to escape the backslash (`\\` vs `/`). ;)

Reply:
Made it a variable. Since this is a .bat file, I don't think I need to worry about escaping backslashes. :)

Reply:
even with batch you still need to be safe, especially with variables, but we're good for now.